cla-backend/cla/routes.py

```
@@ -37,6 +37,14 @@
     get_log_middleware
 )
 
+# Check if authenticated user (via bearer token) is the same as user_id - if not raise exception permission denied
+# LG: comment this out to tunr off this chekc added after LFID is required everywhere in EasyCLA
+def check_user_id_is_current(auth_user, user_id):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)